<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_tarifa</class>
 <widget class="QMainWindow" name="frm_tarifa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tarifas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>411</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Tabela frente Peso Acima do Mínimo</string>
    </property>
    <widget class="QPushButton" name="btn_salvar_taxa">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar Taxas</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Taxas Litoral</string>
     </property>
     <widget class="QLineEdit" name="edt_fpeso_lit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_ad_lit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_gris_lit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_taxa_lit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_icms_lit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Frete Peso</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Ad-valoren</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_44">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> GRIS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_45">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Taxas</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_46">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> ICMS</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Taxas Base</string>
     </property>
     <widget class="QLineEdit" name="edt_icms">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_41">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> GRIS</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_taxa">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_ad">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_38">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Ad-valoren</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_43">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> ICMS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Frete Peso</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_gris">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edt_fpeso">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_42">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Taxas</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gBox_taxasMininas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>471</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Valores dos Fretes Mínimos</string>
    </property>
    <widget class="QLineEdit" name="edt_ad_gris_250">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>190</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_100">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedagio_50">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_lit_200">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedagio_100">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_ad_gris_150">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_lit_20">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_250">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_200">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_lit_50">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_20">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedagio_20">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_lit_100">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedagio_150">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_ad_gris_200">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>160</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_ad_gris_50">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_lit_300">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_150">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedagio_200">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_50">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_ad_gris_100">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedagio_250">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedagio_300">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>220</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_ad_gris_20">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_lit_150">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_ad_gris_300">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>220</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_lit_250">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_base_300">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> Tarifa Base</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_48">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> AD-V/GRIS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_49">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>64</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tarifa Litoral</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_50">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pedagio</string>
     </property>
    </widget>
    <widget class="QLabel" name="desc_20">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>De 01 até 20Kg</string>
     </property>
    </widget>
    <widget class="QLabel" name="desc_150">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>De 101 até 150Kg</string>
     </property>
    </widget>
    <widget class="QLabel" name="desc_100">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>De 51 até 100Kg</string>
     </property>
    </widget>
    <widget class="QLabel" name="desc_300">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>De 251 até 300Kg</string>
     </property>
    </widget>
    <widget class="QLabel" name="desc_50">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>De 21 até 50Kg</string>
     </property>
    </widget>
    <widget class="QLabel" name="desc_250">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>De 201 até 250Kg</string>
     </property>
    </widget>
    <widget class="QLabel" name="desc_200">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>De 151 até 200Kg</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_salvar_tabela">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>260</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar Tabelas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_53">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ped.Litoral</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedLitoral_150">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedLitoral_20">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedLitoral_50">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedLitoral_100">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedLitoral_250">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>190</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedLitoral_300">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>220</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_pedLitoral_200">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>160</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>482</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edt_base_20</tabstop>
  <tabstop>edt_base_lit_20</tabstop>
  <tabstop>edt_ad_gris_20</tabstop>
  <tabstop>edt_pedagio_20</tabstop>
  <tabstop>edt_base_50</tabstop>
  <tabstop>edt_base_lit_50</tabstop>
  <tabstop>edt_ad_gris_50</tabstop>
  <tabstop>edt_pedagio_50</tabstop>
  <tabstop>edt_base_100</tabstop>
  <tabstop>edt_base_lit_100</tabstop>
  <tabstop>edt_ad_gris_100</tabstop>
  <tabstop>edt_pedagio_100</tabstop>
  <tabstop>edt_base_150</tabstop>
  <tabstop>edt_base_lit_150</tabstop>
  <tabstop>edt_ad_gris_150</tabstop>
  <tabstop>edt_pedagio_150</tabstop>
  <tabstop>edt_base_200</tabstop>
  <tabstop>edt_base_lit_200</tabstop>
  <tabstop>edt_ad_gris_200</tabstop>
  <tabstop>edt_pedagio_200</tabstop>
  <tabstop>edt_base_250</tabstop>
  <tabstop>edt_base_lit_250</tabstop>
  <tabstop>edt_ad_gris_250</tabstop>
  <tabstop>edt_pedagio_250</tabstop>
  <tabstop>edt_base_300</tabstop>
  <tabstop>edt_base_lit_300</tabstop>
  <tabstop>edt_ad_gris_300</tabstop>
  <tabstop>edt_pedagio_300</tabstop>
  <tabstop>btn_salvar_tabela</tabstop>
  <tabstop>edt_fpeso</tabstop>
  <tabstop>edt_ad</tabstop>
  <tabstop>edt_gris</tabstop>
  <tabstop>edt_taxa</tabstop>
  <tabstop>edt_icms</tabstop>
  <tabstop>btn_salvar_taxa</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
